version.gradle.kts

```diff
@@ -24,11 +24,11 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-val baseVersion: String by extra("2.0.0-SNAPSHOT.114")
-val coreVersion: String by extra("2.0.0-SNAPSHOT.116")
+val baseVersion: String by extra("2.0.0-SNAPSHOT.115")
```

Comment:
Looking at evolving `Spine` object, I wonder what's the point of maintaining both these versions of Spine libraries along with those defined by `Spine`. Such an approach is very much prone to errors, considering how often our libs are updated. 

Previously, I have encountered some issues caused by using different sources for the versions, in the scope of a single repository. I.e., in one use-case, the version from `versions.gradle.kts` was used, and in another use-case it was grabbed from `Spine` object. And they were different. Took me a couple of hours to figure out, back then.

Please consider addressing this discrepancy.

Reply:
I think, we'd better get rid of all those constants in the `version` file in favor of using default parameters of the Kotlin `Spine` dependency object.

The implementation of `Spine` is currently tricky. Not to address in this PR, but we need it much simpler than now.

Reply:
Added issue: https://github.com/SpineEventEngine/config/issues/418.

Reply:
I've removed constants from `version.gradle.kts` in this PR. But we need more work on this. The added issue is about it.